src/coreclr/vm/appdomain.hpp

```diff
@@ -1217,14 +1217,31 @@ class BaseDomain
     friend class LoadLockHolder;
 public:
     void InitVSD();
-    RangeList *GetCollectibleVSDRanges() { return &m_collVSDRanges; }
+    RangeList *GetCollectibleVSDRanges() { return &m_collVSDRanges; }    
 
 private:
     TypeIDMap m_typeIDMap;
     // Range list for collectible types. Maps VSD PCODEs back to the VirtualCallStubManager they belong to
     LockedRangeList m_collVSDRanges;
 
+#ifdef HOST_WINDOWS
```

Comment:
This will be needed for non-Windows too. (It is ok if you would like to have it under ifdefs for now.)